openedx_external_enrollments/external_enrollments/mit_hz_external_enrollment.py

```diff
@@ -31,6 +32,39 @@ def __init__(self):
     def __str__(self):
         return 'mit_hz'
 
+    def _execute_post(self, url, data=None, headers=None, json_data=None):
+        """
+        Execute post request which is in charge to refresh the user subscription,
+        in case the user does not exist, it will return 404 but the enrollment/subscription
+        should be saved whether it is ok or not in order to track all enrollments.
+        """
+        user_subscription = {}
+        response = requests.post(
```

Comment:
This is the same as calling the base class `_execute_post`, avoiding code duplication:

```suggestion
        response = super()._execute_post(url, data, header, json_data)
```
Then continue checking the response and your custom logic.

Reply:
Nice catch @diegomillan! thanks.